Tools/TestWebKitAPI/Tests/WebKitCocoa/SiteIsolation.mm

```diff
@@ -1825,6 +1825,9 @@ HTTPServer server({
     } };
     for (auto& offsets : selectionOffsetsForFrames)
         findStringAndValidateResults(webView.get(), offsets);
+    findConfiguration.get().backwards = YES;
+    for (auto it = selectionOffsetsForFrames.rbegin() + 1; it != selectionOffsetsForFrames.rend(); ++it)
```

Comment:
Why +1?

Reply:
The last result will be the position we are starting the backwards traversal from, so we should expect the following one.